http/1.6/modules/ROOT/pages/http-listener-ref.adoc

```diff
@@ -101,17 +159,80 @@ This is useful for restricting the visibility of data: you might allow certain u
 </flow>
 ----
 
-When multiple listeners are defined, requests are routed to the first listener matching the method, so default listeners should always be defined last.
+== Configure Response Streaming Mode
 
-=== From HTTP Request to Mule Message
+When HTTP Connector manages response bodies, the connector considers the type of data to send and uses _chunked_ encoding when the body size is not clear. This behavior is also known as streams with no size information.
```

Comment:
HOWEVER, looking at the following text, I think that this might be closer to what you mean:
```suggestion
When HTTP Connector manages response bodies, the connector considers the type of data to send and uses _chunked_ encoding when the body size is not clear. This causes streaming unlimited by data size.
```
...which results in <some negative result>?